### ISSUE TITLE
Tracee is not able to capture all the events

### ISSUE THREAD

**Issue by rohitburke** [aquasecurity/tracee#3163]
## Description
Recently, I noticed one abnormal behavior by Tracee. While performing some action related to networks _(Sending a large number of IPv4 packets)_ and running Tracee behind I saw that there were some **security_socket_connect** and **security_socket_accept** events which got triggered. Okay, but it was quite strange for me that no **network_packets** type of events got captured here because I was dealing with some IPv4 and IPv6 networks. So here I thought why not use the filter option of **net_packet_ipv4** and see if it does trigger that event or not?

Surprisingly, after performing the same action _(Sending a large number of IPv4 packets_) this time **net_packet_ipv4** event got triggered. So is this here _Tracee fail to capture those net_packets events the first time?_ **(when no filter was attached**). 

You can see in the below reference image _**first time I ran Tracee without any filter**_ and later on **I added the filter** you can see the difference. Let me know If there is something wrong with my side.

![bug](https://github.com/aquasecurity/tracee/assets/30924877/f13d468e-e8df-420b-b33e-b154c7893dcc)

## Output of `tracee version`:

```
v0.14.2
```

## Output of `uname -a`:

```
Linux kali 6.1.0-kali7-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.20-2kali1 (2023-04-18) x86_64 GNU/Linux
```

**Comment by yanivagman** [comment]
So the thing is that `net_packet_ipv4` is not part of the default set of evets used by Tracee.
You can use the --list command to list all the events supported by Tracee, then grep for `default` to only list the events in the default set:
`./dist/tracee list | grep default`

Also note that the above command is using the new unified `tracee` binary, while you are using the now deprecated `tracee-ebpf` binary

**Comment by rohitburke** [comment]
Why does it's not included in the default set of events? So you mean every time whenever I want to see network packet events going on in the background I need to add a filter first?

**Comment by rohitburke** [comment]
Okay, this is understandable, but suppose a scenario comes where I'm running Tracee and I am also running an application which have some interaction with networks. So what I will do is I will add the filter or **net_packet_ipv4** or **net_packet_ipv6** (And I will get events based on it referred to (1) in image below.)

So this way I will get to know how that application is interacting. But suppose if the same application try to interact with my server using HTTP request or by DNS request. Tracee will capture that when we add those filter events in the cmd line (As referred to (2) in the image below)  

![Screenshot 2023-05-30 152331](https://github.com/aquasecurity/tracee/assets/30924877/29dacda0-e940-40ce-a502-bb06f844cbf8)

Now, as a normal user how you can know that this type of malicious application will interact to our server via HTTP or DNS request. Why as a user you will add this event (net_packet_dns_request and net_packet_http_request) and trace your application.

Think that how that user will get to know that this application is using such request (**condition when he will not add the filter of HTTP or DNS). Now, If you will say me the solution is to add all the network events in the filter events but what if in the future we have 30 to 40 events in that case why we will be adding the events in command line. 

I hope you understand my point.

**Comment by yanivagman** [comment]
It's for the user's decision to decide which kind of events is of interest to him.
If you are interested both in DNS and HTTP events, then simply choose those.
By the way, we recently introduced the concept of policies of Tracee - are you familiar with that?
It makes the procedure of selecting which events should be traced for different scopes (so you can actually trace different events for different workloads). See here: https://aquasecurity.github.io/tracee/v0.14/docs/policies/

**Comment by rohitburke** [comment]
Yes, I had a look into these policies. 
As you said it's **the user's decision to decide which kind of events interests him**.
That's correct, but do you know where I faced the issue?

I decided to monitor one application and for that I use TRACEE. I ran Tracee in the background and I perform some action on that application. I saw Tracee's output it shows me several events. This is what normal things I do to monitor any application.

But then I asked myself a question why TRACEE is not giving me any network events despite the fact that the application interacts with the internet? Then I filter the network events and I was able to see them. So, think here if any security team is monitoring any such application and they decide not to use any filter and run tracee as it is. Then they will not be able to see those events.

_So my point is on NORMAL monitoring (we should able to see such important events.)_
The point which you told me was **It's for the user's decision to decide which kind of events is of interest to him.**
_But here user just wants to see(monitor) all the events, Here user doesn't want to select and see any specific events._

**What [#2611](https://github.com/aquasecurity/tracee/discussions/2611) here we discuss maybe true at some instances but I personally feel that TRACEE should disable only those kinds of events that are related to the daemon or some other background OS events, disabling important network-like events may not be a good option.**

**Comment by yanivagman** [comment]
So what you suggest is to change the default set of events, right?
Specifically the net_packet_ipv4 event is just too noisy. This event happens on every packet sent or received, and will make the event log explode fast on most environments where you have some network traffic going on.
We could have added the DNS and HTTP events, but these use cgroup based programs and some hook points that are only available from kernel 5.4. As tracee currently supports 4.19 kernels, we opted to avoid adding these events to the default set of events. A user that have a kernel that supports these events can simply add them to the default set (by choosing the default set and these extra events).

**Comment by rohitburke** [comment]
Ohh, I understand that no problem may be in the future we can decide what are the best options available to get all events in one place. We can close this issue. @yanivagman
